src/modules/LimitOrders.ts

```diff
@@ -325,23 +324,23 @@ export class LimitOrders {
   public getOrderHash(
     order: LimitOrder,
   ): string {
-    const structHash = soliditySha3(
+    const structHash = this.web3.utils.soliditySha3(
```

Comment:
why are you using both `Web3.utils` and `this.web3.utils` in different places? We should probably just use `Web3.util` everywhere